src/scent_block.h

```diff
@@ -70,10 +70,11 @@ struct scent_block {
     }
 
     // We should be working entirely within the range, so don't range check here
-    void apply_gas( const tripoint &p ) {
+    void apply_gas( const tripoint &p, int nintensity = 0 ) {
```

Comment:
```suggestion
    void apply_gas( const tripoint &p, const int nintensity = 0 ) {
```

Reply:
Typo - `nintensity` to `intensity`

Reply:
it's not a typo, the "n" stands for "new," it's a similar thing as to what the mod_stats type functions do, but i don't have a strong attachment to it

Reply:
what's the point in making it const? it's not a reference. i know i don't alter the value though

Reply:
I think clang-tidy will actually yell at you for "const int" now.

Reply:
> what's the point in making it const? it's not a reference. i know i don't alter the value though

It may not be obvious for other maintainers that the value is not changed. Declaring it const allows them to know the value is not changed without analyzing the code in the function.

> I think clang-tidy will actually yell at you for "const int" now.

Only for declarations though. This is an inline definition, so `const int` should be fine.